Explorer/Assets/Scripts/ECS/Unity/Materials/Systems/CreatePBRMaterialSystem.cs

```diff
@@ -32,20 +32,23 @@ protected override void Update(float t)
         }
 
         [Query]
-        private void Handle(ref MaterialComponent materialComponent)
+        [All(typeof(ShouldInstanceMaterialComponent))]
+        private void Handle(Entity entity, ref MaterialComponent materialComponent)
         {
             if (!materialComponent.Data.IsPbrMaterial)
                 return;
 
+            // if there are no textures to load we can construct a material right away
```

Comment:
Looks like this comment shouldn't be here, but rather just above 'ConstructMaterial'

Reply:
I moved the corresponding comment as well as the checking of the condition itself